lib/shared/addon/components/authorized-endpoint/component.js

```diff
@@ -68,7 +68,9 @@ export default Component.extend({
 
     const lcae = globalStore.createRecord({
       type:    'localClusterAuthEndpoint',
-      enabled: true
+      enabled: true,
+      fqdn:    '',
+      caCerts: ''
```

Comment:
Why do these need to be set as empty strings instead of leaving them as undefined?